css/builder.css

```
@@ -556,7 +556,15 @@ input[type="time"]:not(.basic-time-picker)::-webkit-calendar-picker-indicator {
 }
 
 .fl-timer {
-  column-gap: 8px;
+  width: 100%;
+  display: flex;
+  column-gap: 10px;
+  margin-top: 5px;
+}
+
+.fl-timer > div{
```

Your IDE should be able to autofix this error when you save the file. Can you confirm your setup is working correctly?

```suggestion
.fl-timer > div {
```